packages/block-library/src/quote/save.js

```diff
@@ -6,21 +6,32 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { RichText, useBlockProps } from '@wordpress/block-editor';
+import { InnerBlocks, RichText, useBlockProps } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { align, value, citation } = attributes;
+	const { align, citation } = attributes;
 
 	const className = classnames( {
 		[ `has-text-align-${ align }` ]: align,
 	} );
 
 	return (
-		<blockquote { ...useBlockProps.save( { className } ) }>
-			<RichText.Content multiline value={ value } />
-			{ ! RichText.isEmpty( citation ) && (
-				<RichText.Content tagName="cite" value={ citation } />
+		<>
+			{ RichText.isEmpty( citation ) ? (
+				<blockquote { ...useBlockProps.save( { className } ) }>
+					<InnerBlocks.Content />
+				</blockquote>
+			) : (
+				<figure { ...useBlockProps.save( { className } ) }>
+					<blockquote>
+						<InnerBlocks.Content />
+					</blockquote>
+
+					<figcaption>
+						<RichText.Content tagName="cite" value={ citation } />
```

Comment:
I don't believe we need to or even necessary _want_ to wrap this whole thing in `<cite>`. The [spec](https://html.spec.whatwg.org/#the-cite-element) provides clear examples of how to use `<cite>` and how _not_ to use it.

Instead we may want to prefer using `<RichText.Content tagName="fig caption" value={ citation } />` and then if we need `<cite>`, to add a format for it for `RichText`